src/components/LikeAndDislikeButtons.js

```diff
@@ -30,14 +30,24 @@ const DislikedIcon = styled(Icon)`
   color: #d32f2f;
 `;
 
-const LikeAndDislikeButtons = () => {
+const LikeAndDislikeButtons = ({ throwCard }) => {
   return (
     <ButtonContainer>
       <IconWrapper>
-        <DislikedIcon className="material-icons">clear</DislikedIcon>
+        <DislikedIcon
+          className="material-icons"
+          onClick={() => throwCard('left')}
```

Comment:
Would be really great if you could apply an animation to the card on click so it looks like it's being thrown out, rather than just disappearing.

Reply:
Having researched this I don't think it is possible

Reply:
Yeah maybe raise it in a `nice to have` issue :+1: